python/packages/ai/teams/ai/utilities.py

```diff
@@ -50,3 +52,59 @@ def to_string(tokenizer: Tokenizer, value: Any, as_json: bool = False) -> str:
         return yaml_str
 
     return json_str
+
```

Comment:
can you move these to `./utils` folder and separate them into individual files please